src/pyconnect/pyconnectsink.py

```diff
@@ -377,8 +378,20 @@ def _commit(self) -> None:
         if not offsets:
             logger.info("No offsets to commit.")
         else:
-            logger.info(f"Committing offsets: {offsets}")
-            self._consumer.commit(offsets=offsets, asynchronous=False)
+            max_attempts: int = 2
+            attempt_count: int = 1
+            while attempt_count <= max_attempts:
+                try:
+                    logger.info(f"Committing offsets: {offsets}")
+                    self._consumer.commit(offsets=offsets, asynchronous=False)
+                except KafkaException as ke:
+                    logger.error(
```

Comment:
```suggestion
                    logger.warn(
```
Since we're retrying, a warning should be sufficient.